modules/oxxionRtdProvider.md

```diff
@@ -23,14 +23,16 @@ Then add the oxxion Rtd module to your prebid configuration :
 pbjs.setConfig(
   ...
   realTimeData: {
-    auctionDelay: 200,
+    auctionDelay: 300,
```

Comment:
typo on line 15 above

Reply:
You meant to add the new use case in the description ?

Reply:
you write rtbModule instead of rtdmodule

Reply:
Thanks, I hadn't seen it. It's fixed.